src/elements/Toast.tsx

```diff
@@ -0,0 +1,94 @@
+import { useDispatch, useSelector } from "react-redux";
+import {
+  SyntheticEvent,
+  useEffect,
+  useState
+} from "react";
+import { clearToastMessage, selectUi } from "modules/ui";
+import { Alert, Snackbar } from "@mui/material";
+import CheckCircleIcon from "assets/images/CheckCircle";
+import CloseCircleFill from "assets/images/CloseCircleFill";
+import styled from "styled-components";
+import theme from "theme";
+
+const StyledHeading = styled.p`
+  margin: 0;
+  font-weight: ${theme.typography.fontWeightBold};
+  font-size: 12px;
+  line-height: 14.52px;
```

Comment:
@scandycuz So based on the latest on typography talks, how should i handle the `font-size` and `line-height` here?